pkg/yurtappmanager/controller/uniteddeployment/adapter/statefulset_adapter.go

```diff
@@ -148,6 +148,32 @@ func (a *StatefulSetAdapter) ApplyPoolTemplate(ud *alpha1.UnitedDeployment, pool
 	set.Spec.VolumeClaimTemplates = ud.Spec.WorkloadTemplate.StatefulSetTemplate.Spec.VolumeClaimTemplates
 
 	attachNodeAffinityAndTolerations(&set.Spec.Template.Spec, poolConfig)
+
+	if poolConfig.Patches == nil {
```

Comment:
The code block here is quite similar to the code in deployment adapter. Can you extract them to a common method?

Reply:
done